userspace/falco/versions_info.cpp

```diff
@@ -49,14 +49,15 @@ static inline std::string get_driver_schema_version(const std::shared_ptr<sinsp>
 }
 
 falco::versions_info::versions_info(const std::shared_ptr<sinsp>& inspector)
+    : falco_version(FALCO_VERSION)
+    , engine_version(FALCO_ENGINE_VERSION)
+    , libs_version(FALCOSECURITY_LIBS_VERSION)
+    , plugin_api_version(inspector->get_plugin_api_version())
+    , driver_api_version(get_driver_api_version(inspector))
+    , driver_schema_version(get_driver_schema_version(inspector))
+    , default_driver_version(DRIVER_VERSION)
+
 {
-    falco_version = FALCO_VERSION;
-    engine_version = FALCO_ENGINE_VERSION;
-    libs_version = FALCOSECURITY_LIBS_VERSION;
-    plugin_api_version = inspector->get_plugin_api_version();
-    driver_api_version = get_driver_api_version(inspector);
-    driver_schema_version = get_driver_schema_version(inspector);
-    default_driver_version = DRIVER_VERSION;
```

Comment:
Good point, would this something to be done in a follow up PR ?

Reply:
> Good point, would this something to be done in a follow up PR ?

The note on the changing the parameter `shared_ptr<sinsp>` to a reference or a pointer? Yes, I'd say it can be addressed later.